content/discover/in-the-media.adoc

```diff
@@ -9,7 +9,7 @@ title = "In the Media"
 
 **KiCad has been featured in the following blog posts, press and media:**
 
-https://www.digikey.com/en/blog/interview-with-wayne-stambaugh-pt1[Interview with Wayne Stambaugh about KiCad, Part 1] (13-MAR-2018) https://www.digikey.com/en/blog/interview-with-wayne-stambaugh-pt2[Part 2] (19-MAR-2018)
+https://www.digikey.com/en/blog/interview-with-wayne-stambaugh-pt1[Interview with Wayne Stambaugh about KiCad, Part 1] (13-MAR-2018) https://www.digikey.com/en/blog/interview-with-wayne-stambaugh-pt2[Part 2] (19-MAR-2018) https://www.digikey.com/en/blog/interview-with-wayne-stambaugh-pt2[Part 3] (26-MAR-2018)
```

Comment:
I spot a mistake here :P But thank you.

Reply:
https://github.com/KiCad/kicad-website/commit/e71629278bba809bd7e50dcc4c742baba9933bff

Reply:
ugh, sorry...